packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1182,6 +1182,11 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
     return 0.0;
   }
 
+  @override
+  bool doesAcceptAdjustedPositionForNewDimensions(double adjustedPixels) {
+    return false;
+  }
```

Comment:
assuming doesAcceptAdjustedPositionForNewDimensions exists just for this, could we instead just have the NestedScrollView logic pass in a ScrollPhysics class that overrides adjustPositionForNewDimensions?

Reply:
I was following your lead, you had a similar method on ScrollPosition.  I thought your intention was to allow people to customize there as well.  I can revisit this once I get the baseline functionality.